pallets/aggregate/src/should.rs

```diff
@@ -627,7 +632,9 @@ mod aggregate {
                 Aggregate::on_proof_verified(Some(account), DOMAIN, statement);
             }
 
-            let expected_balance = Balances::free_balance(USER_DELIVERY_OWNER) + DELIVERY_PRICE;
+            let delivery_per_statement = (DELIVERY_FEE + OWNER_TIP) / DOMAIN_SIZE as u128;
+            let total_collected = delivery_per_statement * DOMAIN_SIZE as u128;
+            let expected_balance = Balances::free_balance(USER_DELIVERY_OWNER) + total_collected;
 
             assert_ok!(Aggregate::aggregate(
```

Comment:
Not sure if it's a test to be placed in this pallet or somewhere else, but we should also check that (in case of Hyperbridge for instance) the delivery owner actually pays for the delivery, i.e. that after the aggregate() call the delivery owner will have only it's initial_balance + tip (the delivery cost was covered by proof submitters)